.mci.yml

```diff
@@ -106,6 +106,7 @@ functions:
                  g++ --version || true
                  clang --version || true
                  cmake --version || true
+                 openssl version || true
```

Comment:
This helped debug the incompatible cmake and openssl version issue

Reply:
Acknowledged